serpent-moltres/full-core/postprocessing.py

```diff
@@ -192,3 +349,235 @@ def plot_radial_power_distribution(power, save):
+    * radial flux at 600 K and 1200 K
+    * radial power distribution at 600 K and 1200 K
+    '''
+
+    G = 15
+    lim = choose_lim(G)
+
+    # axial flux at 600 K
+    file = '3D-fullcore-600-' + str(G) + 'Gd_axial1_0002.csv'
+    save = '3D-fullcore-600-' + str(G) + 'Gd-axial1'
+    plt.figure()
+    plotcsv_groups_collapse(file, save, lim, dire='z')
+    plt.savefig(save, dpi=300, bbox_inches="tight")
+    plt.close()
+
+    # radial flux at 600 K
+    file = '3D-fullcore-600-' + str(G) + 'Gd_radial1_0002.csv'
+    save = '3D-fullcore-600-' + str(G) + 'Gd-radial1'
+    plt.figure()
+    plotcsv_groups_collapse(file, save, lim, dire='r')
+    plt.savefig(save, dpi=300, bbox_inches="tight")
+    plt.close()
+
+    # axial flux at 600 K
+    file = '3D-fullcore-1200-' + str(G) + 'Gd_axial1_0002.csv'
+    save = '3D-fullcore-1200-' + str(G) + 'Gd-axial1'
+    plt.figure()
+    plotcsv_groups_collapse(file, save, lim, dire='z')
+    plt.savefig(save, dpi=300, bbox_inches="tight")
+    plt.close()
+
+    # radial flux at 600 K
+    file = '3D-fullcore-1200-' + str(G) + 'Gd_radial1_0002.csv'
+    save = '3D-fullcore-1200-' + str(G) + 'Gd-radial1'
+    plt.figure()
+    plotcsv_groups_collapse(file, save, lim, dire='r')
+    plt.savefig(save, dpi=300, bbox_inches="tight")
+    plt.close()
+
+    # radial power distribution at 600 K
+    filename = '3D-fullcore-600-' + str(G) + 'Gd.csv'
+    file = pd.read_csv(filename)
+    power = np.zeros(11)
+    for i in range(11):
+        power[i] = float(file['F' + str(i+1) + '_fission_heat'].tolist()[-1])
+    power /= 1e6  # W to MW
+
+    save = '3D-fullcore-600-' + str(G) + 'Gd-power'
+    plot_radial_power_distribution(power, save)
+
+    # radial power distribution at 1200 K
+    filename = '3D-fullcore-1200-' + str(G) + 'Gd.csv'
+    file = pd.read_csv(filename)
+    power = np.zeros(11)
+    for i in range(11):
+        power[i] = float(file['F' + str(i+1) + '_fission_heat'].tolist()[-1])
+    power /= 1e6  # W to MW
+
+    save = '3D-fullcore-1200-' + str(G) + 'Gd-power'
+    plot_radial_power_distribution(power, save)
+
+
+def compare_serpent_moltres():
+    '''
+    Obtains figures w/ the results from Serpent and Moltres simulations.
+    The figures contain the following information:
+    * axial fluxes at 600 K and 1200 K
+    * radial fluxes at 600 K and 1200 K
+    * radial power distributions at 600 K and 1200 K
+    '''
+
+    # radial power distribution at 600 K
+    data = st.read('oecd-fullcore-600_det1b1.m', reader='det')
+    det = data.detectors['power']
+    power = np.zeros(11)
+    power[0] = det.tallies[0, 3]
+    power[1] = det.tallies[0, 4]
+    power[2] = det.tallies[1, 2]
+    power[3] = det.tallies[1, 3]
+    power[4] = det.tallies[1, 4]
+    power[5] = det.tallies[2, 1]
+    power[6] = det.tallies[2, 2]
+    power[7] = det.tallies[2, 3]
+    power[8] = det.tallies[3, 1]
+    power[9] = det.tallies[3, 2]
+    power[10] = det.tallies[4, 1]
```

Comment:
Hardcoding numbers like this with liberal copy-pasting is the easiest way to create an incomprehensible bug. Please use loop logic. DRY